.github/actions/constellation_create/action.yml

```diff
@@ -211,8 +211,8 @@ runs:
       shell: bash
       run: |
         constellation init --force --debug
-        echo "KUBECONFIG=$(pwd)/constellation-admin.conf" >> $GITHUB_OUTPUT
-        echo "MASTERSECRET=$(pwd)/constellation-mastersecret.json" >> $GITHUB_OUTPUT
+        echo "KUBECONFIG=$(pwd)/constellation-admin.conf" | tee -a $GITHUB_OUTPUT
+        echo "WORKSPACE=$(pwd)" | tee -a $GITHUB_OUTPUT
```

Comment:
```suggestion
```
The workspace during create should be the same during create and all other actions, I don't think we need to specifically set it just for recover